src/v1/database_admin_client_config.json

```diff
@@ -118,6 +118,10 @@
           "timeout_millis": 3600000,
           "retry_codes_name": "idempotent",
           "retry_params_name": "e9fafda332ce8a1702dc1575de3ca81c4feb4799"
+        },
+        "ListDatabaseRoles": {
+          "retry_codes_name": "non_idempotent",
```

Comment:
I know that this is auto-generated, so not something that we can change here, but maybe you can put the question to the team that created the proto changes: Why is this `non_idempotent`? I would expect this method to be idempotent.

Reply:
This is autogenerated. But I can check and find out